Android/app/src/main/java/com/example/boardgamesocial/LoginAndSignUp/Fragments/SignUpFragment.java

```diff
@@ -72,8 +72,9 @@ private void registerUser() {
         String username = Objects.nonNull(etUsername.getText()) ? etUsername.getText().toString() : null;
         String password = Objects.nonNull(etPassword.getText()) ? etPassword.getText().toString() : null;
 
-        retrofitClient.signUpCall(new User(firstName, lastName, email, username, password)).subscribe(user -> {
+        retrofitClient.signUpCall(new User(firstName, lastName, email, username, password)).blockingSubscribe(user -> {
```

Comment:
Why are you using blockingSubscribe here?

Reply:
Before the signup functionality wasn't working, that's the reason I added it but if it's working and not needed we can remove it